insights/parsers/mongod_conf.py

```diff
@@ -0,0 +1,93 @@
+"""
+mongodb-server configuration files
+==================================
+
+This module contains the following files:
+    ``/etc/mongod.conf`` and ``/etc/mongodb.conf``.
+
+Both of them are provided by package mongodb-server.
+This MongoDB configuration file may use the **YAML** format
+or the standard **key-value pair** format.
+
+
+Sample input(YAML format)::
+
+    systemLog:
+      destination: file
+      logAppend: true
+      path: /var/log/mongodb/mongod.log
+
+    # Where and how to store data.
+    storage:
+      dbPath: /var/lib/mongo
+      journal:
+        enabled: true
+
+
+Sample input(key-value pair format)::
+
+    # mongodb.conf - generated from Puppet
+    #where to log
+    logpath=/var/log/mongodb/mongodb.log
+    logappend=true
+    # Set this option to configure the mongod or mongos process to bind to and
+    # listen for connections from applications on this address.
+    # You may concatenate a list of comma separated values to bind mongod to multiple IP addresses.
+    bind_ip = 127.0.0.1
+    # fork and run in background
+    fork=true
+    dbpath=/var/lib/mongodb
+    # location of pidfile
+    pidfilepath=/var/run/mongodb/mongodb.pid
+    # Enables journaling
+    journal = true
+    # Turn on/off security.  Off is currently the default
+    noauth=true
+
+
+Examples:
+
+    >>> mongod_conf = shared[MongodConf]
+    >>> 'storage' in mongod_conf.data
```

Comment:
Since "LegacyItemAccess" is used, better remove the ".data" from example.

Reply:
Done.